pages/organization-onboarding.tsx

```diff
@@ -43,8 +47,9 @@ interface OrganizationOnboardingPageProps {
 
 const OrganizationOnboardingPage: React.FC<OrganizationOnboardingPageProps> = () => {
   const intl = useIntl()
-  const stats = useSelector((state: RootState) => state.startup.stats)
-  const volunteerCountChars = String(stats.volunteers).split('')
+  const { stats, loading } = useChannelStats()
+  const volunteerCountChars = String(stats?.volunteersCount).split('')
```

Comment:
```suggestion
  const volunteerCountChars = stats ? String(stats.volunteersCount).split('') : ''
```

Aqui se o stats fosse === undefined, a string resultante seria "undefined" e causaria o resultado ser [u][n]...